apps/dataset-ingestion/app/build_faces/create_descriptorsets.py

```diff
@@ -8,5 +8,5 @@
 }
 for set_name, dim in set_names.items():
     utils.add_descriptorset(set_name, dim,
-            metric=["L2"],
-            engine=["FaissFlat"])
+            metric=["L2", "CS", "IP"],
```

Comment:
unless you select which one to use on the query side, having all these indexes (6 in total) may be slower to build.

Reply:
I shall time the difference, and if there's a gap, will figure out the right metric here. We already know the CLIP metric. need to check the same when using Facenet.